rts/at/util/src/main/java/org/jboss/jbossts/star/util/media/txstatusext/TransactionManagerElement.java

```diff
@@ -33,7 +33,7 @@
 
     @XmlElement
     public Date getCreated() {
-        return created;
+        return new Date(created.getTime());
```

Comment:
If this is really about mutability, low about returning the long instead of a new object (I am thinking perf)